publication/src/main/scala/hmda/publication/reports/AggregateAndDisclosureReports.scala

```diff
@@ -0,0 +1,28 @@
+package hmda.publication.reports
+
+import akka.actor.{ Actor, ActorRef, ActorSystem, Props }
+import akka.stream.ActorMaterializer
+import hmda.publication.reports.disclosure.DisclosureReports
+
+object AggregateAndDisclosureReports {
+  case class GenerateDisclosureByMSAReports(respondentId: String, fipsCode: Int)
+  def props(): Props = Props(new AggregateAndDisclosureReports)
+  def createAggregateDisclosureReports(system: ActorSystem): ActorRef = {
+    system.actorOf(AggregateAndDisclosureReports.props())
+  }
+}
+
+class AggregateAndDisclosureReports extends Actor {
+
+  import AggregateAndDisclosureReports._
+
+  implicit val system = context.system
+  implicit val ec = system.dispatcher
+  implicit val materializer = ActorMaterializer()
+
+  override def receive: Receive = {
+    case GenerateDisclosureByMSAReports(respId, fipsCode) =>
```

Comment:
Totally. I made an issue to remind us: #1071. Feel free to add any other actors that need testing.